src/java.security.jgss/share/classes/sun/security/krb5/Checksum.java

```diff
@@ -73,56 +73,7 @@
     // draft-brezak-win2k-krb-rc4-hmac-04.txt
     public static final int CKSUMTYPE_HMAC_MD5_ARCFOUR = -138;
 
```

Comment:
nit: How about updating and fixing the copyright years?

Reply:
Will update it in the next commit.